docs/resources/cluster.md

```diff
@@ -1476,6 +1476,7 @@ The following arguments are supported:
 * `compartment_id` - (Required) The OCID of the compartment in which to create resources OKE cluster and related resources (string)
 * `custom_boot_volume_size` - (Optional) Optional custom boot volume size (GB) for all nodes. If you specify 0, it will apply the default according to the `node_image` specified. Default `0` (int)
 * `description` - (Optional) An optional description of this cluster (string)
+* `enable_private_control_plane` - (Optional) Specifies whether Kubernetes API endpoint is a private IP only accessible from within the VCN. Default `false` (bool)
```

Comment:
This new argument is just gonna work on next Rancher releases, let's say Rancher v2.5.10. Could you please add a comment here to specify that the argument should work just for Rancher v.2.5.10 and above??

Reply:
Sure will do.  Good suggestion.